### PR TITLE
Replace Account with Keypair

### LINKED ISSUE #62: Deprecate Account
Deprecate Account for Keypair: https://solana-labs.github.io/solana-web3.js/classes/account.html

### PR DESCRIPTION
**Pull request by kevinheavey**
Closes #62 

Making this a draft PR because I'll want to look again tomorrow, but would like to hear thoughts in the meantime.

Summary of changes:

- Add keypair.py similar to keypair.ts
- Replace all usage of Account with Keypair, except in account.py itself
- Add deprecation warning to Account

### CODE REVIEW

File: solana/rpc/_utils/encoding.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def _json_list_errors(self, iterable: Iterable[Any]) -> Iterable[str]:
             try:
```
Comment (michaelhly) [inline solana/rpc/_utils/encoding.py:26]: Can we make these changes in a separate PR?

**Review by michaelhly — COMMENTED** [review]
Small nit.

File: solana/rpc/_utils/encoding.py (hunk 1 above)
Comment (kevinheavey) [reply solana/rpc/_utils/encoding.py]: (Un)done

**Review by michaelhly — APPROVED** [review]
